client/src/_assets/styles/payment-table-overrides.css

```diff
@@ -60,4 +60,12 @@
 
 .header-text {
     color: #004A6E;
+}
+
+.action-icon {
+    font-size: 12px;
+}
+
+.actionsButton {
```

Comment:
this is a nitpick, but could you change this class name to `actions-button` to keep with naming conventions for class names?